### PR TITLE
VACMS-3147: Updates Drush Site Status commands for Drush 10.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #3147.

I'll test this on other environments when the PR's fully open, but these commands are pretty trivial.

This commit doesn't change the `composer.json` or `composer.lock`, just adds the new files.  I'm going to check in the next little bit to see if these added files interfere with Drush 8.

I looked and didn't see any code or scripts using `va-gov-get-deploy-mode` programmatically, but I might've missed some, and the terseness of the response (`ENABLED` or `DISABLED`) made me think that this might be used somewhere in a script, so I simply `echo`ed that string to output rather than rephrasing it in a friendlier way.

## Testing done
Ran the commands, saw expected output.  Since these are simple passthrough commands, I don't see much value in automated tests.

## Screenshots
Screen Shot 2021-03-03 at 10 07 29 AM
Screen Shot 2021-03-03 at 10 16 20 AM

## QA steps
As a developer running Drush 10:
1. Check `lando drush list` and verify that:
    - [ ] the commands appear in the list and have appropriate descriptions.
2. Run the commands and verify that the results are as expected:
    - [ ] `va-gov-get-deploy-mode` should return either `DISABLED` or `ENABLED` as appropriate, as the Drush 8 command did
    - [ ] `va-gov-enable-deploy-mode` should enable deploy mode
    - [ ] `va-gov-disable-deploy-mode` should disable deploy mode

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ndouglas** [comment]
Output of previous version for the sake of comparison -- Screen Shot 2021-03-03 at 10 27 27 AM

Gonna go ahead and inject that newline (still getting used to `zsh`).

File: docroot/modules/custom/va_gov_build_trigger/composer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "name": "org/va_gov_build_trigger",
+    "type": "drupal-drush",
+    "extra": {
+        "drush": {
+            "services": {
+                "drush.services.yml": "^10"
+            }
+        }
+    }
+}
```
Comment (ndouglas) [inline docroot/modules/custom/va_gov_build_trigger/composer.json:11]: I'ma remove this.

Comment (ndouglas) [reply docroot/modules/custom/va_gov_build_trigger/composer.json]: Screen Shot 2021-03-03 at 12 43 41 PM

Confirmed that removing it didn't introduce any issues.

**Comment by acrollet** [comment]
looking good:

Screen Shot 2021-03-03 at 12 21 40 PM

**Review by acrollet — APPROVED** [review]
QA steps pass, code looks clean, approved. One curiosity: is there a drush-ish that we can/should use with `getDeployMode()` instead of `echo`?

**Comment by ndouglas** [comment]
Given the nature of the response for that command, I figured it'd be better to return as raw a response as possible.  I don't really have a problem with passing this through the Drush logger, but don't see much of a reason to do it either.

**Comment by acrollet** [comment]
> Given the nature of the response for that command, I figured it'd be better to return as raw a response as possible. I don't really have a problem with passing this through the Drush logger, but don't see much of a reason to do it either.

👍 just was curious

**Comment by indytechcook** [comment]
FYI, I don't think that command is used in any scripts currently.

**Review by indytechcook — APPROVED** [review]

**Pull request merged**